idv/mobile-swift-ui/mobile-swift-ui/OnboardingComponents/EmailAndPhoneView.swift

```diff
@@ -1,67 +1,152 @@
+                                FpInput(placeholder: "Enter your email")
+                                    .padding()
+                                    .background(Color.gray.opacity(0.1))
+                                    .cornerRadius(10)
+                                FpFieldError()
+                            }
+                        }
+                    )
+                    
+                    FpField(
+                        name: .idPhoneNumber,
+                        content: {
+                            VStack(alignment: .leading){
+                                FpLabel("Phone Number", font: .subheadline, color: .secondary)
+                                FpInput(placeholder: "Enter your phone number")
+                                    .padding()
+                                    .background(Color.gray.opacity(0.1))
+                                    .cornerRadius(10)
+                                FpFieldError()
+                            }
+                        }
+                    )
+                    Button(action: formUtils.handleSubmit) {
+                        if isLoading {
+                            ProgressView() // Loading indicator
+                                .progressViewStyle(CircularProgressViewStyle())
+                                .frame(width: 100, height: 50)
+                                .background(Color.blue)
+                                .cornerRadius(10)
+                                
+                        } else {
+                            Text("Submit")
+                                .font(.headline)
+                                .foregroundColor(.white)
+                                .frame(width: 100, height: 50)
+                                .background(Color.blue)
+                                .cornerRadius(10)   
+                        }
                     }
+                    .disabled(isLoading)
                 }
-                .foregroundColor(.white)
-                .padding()
-                .frame(maxWidth: .infinity)
-                .background(isLoading ? Color.gray : Color.blue)
-                .cornerRadius(8)
-                .disabled(isLoading)
-                .padding()               
-                NavigationLink(destination: VerifyOTPView(), isActive: $shouldNavigateToNextView) { EmptyView() }
             }
-        }
+            
+        )
+        .padding(.horizontal, 20)
         .navigationTitle("Signup flow")
-        .onAppear(perform: {
+        .onAppear {
             Task {
                 do {
                     let sandboxOutcome = SandboxOutcome(overallOutcome: .pass, documentOutcome: .pass)
                     try await FootprintProvider.shared.initialize(
-                        configKey: "pb_test_QeSAeS8XHohiSpCOj2l4vd",
-//                        sandboxId: "gC2hvdsN06Q53",
+                        configKey: "pb_test_OmuMuP68H00rL3vVxRWg6f",
```

Comment:
Let's use a key that doesn't require passkey and id-doc to support inline process